src/rdiff_backup/restore.py

```diff
@@ -441,6 +444,11 @@ def __init__(self, mirror_rp, inc_rp, inc_list):
         self.inc_rp, self.inc_list = inc_rp, inc_list
         self.set_relevant_incs()
 
+    def __str__(self):
```

Comment:
That was used for debugging ?

Reply:
Yes, it's quite handy in the debugger, you just call `p str(rf_object)` and you get an overview of the object's content. It's nowhere used in the code.